tests/test_views_permissions.py

```diff
@@ -19,6 +19,44 @@ def get_app_settings(cls, extras=None):
         return settings
 
 
+class PermissionsUnauthenticatedViewTest(BaseWebTest, unittest.TestCase):
+
+    @classmethod
+    def get_app_settings(cls, extras=None):
+        settings = super().get_app_settings(extras)
+        settings['experimental_permissions_endpoint'] = 'True'
+        settings['bucket_create_principals'] = 'system.Everyone'
+        return settings
+
+
+class EntriesTestUnauthenticated(PermissionsUnauthenticatedViewTest):
```

Comment:
You don't need that class, you can just put the test into `PermissionsUnauthenticatedViewTest`

Reply:
Thanks, this was my first time doing real testing, I'll clean up all of that.